src/Parser/DeathKnight/Unholy/Modules/Features/ScourgeStrikeEfficiency.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import { formatPercentage } from 'common/format';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Enemies from 'Parser/Core/Modules/Enemies';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+class ScourgeStrikeEfficiency extends Analyzer {
+  static dependencies = {
+    enemies: Enemies,
+    combantants: Combatants,
+  };
+
+  // used to track how many stacks a target has
+  targets = {};
+
+  totalScourgeStrikeCasts = 0;
+  scourgeStrikeCastsZeroWounds = 0;
+
+  on_byPlayer_applydebuffstack(event){
+    const spellId = event.ability.guid;
+    if(spellId === SPELLS.FESTERING_WOUND.id){
+      this.targets[event.targetID] = event.stack;
+    }
+  }
+
+  on_byPlayer_removedebuffstack(event){
+    const spellId = event.ability.guid;
+    if(spellId === SPELLS.FESTERING_WOUND.id){
+      this.targets[event.targetID] = event.stack;
```

Comment:
You should probably check for targetID **AND** targetInstance, since there can be multiple mobs with the same ID up at any given moment